app/pb_kits/playbook/pb_textarea/textarea.rb

```diff
@@ -7,33 +7,45 @@ class Textarea
 
       partial "pb_textarea/textarea"
 
-      prop :error
-      prop :object
+      prop :error, type: Playbook::Props::Boolean,
+                   default: false
+      prop :error_message, type: Playbook::Props::String,
+                           default: "This field has an error."
+      prop :input_id
```

Comment:
All your changes look great!

This is the last bit. I think an `input_id` prop is a bad pattern because it makes sense to pass anything that goes on the input to the `input_options` prop. That is how this is accomplished on PbCheckbox, PbToggle, PbRadio, PbSelectableCard, PbSelectableIcon, and PbSelectableCardIcon (some of those are yet to be merged). I know we have a couple kits with inputs that use this prop, but I'd like to deprecate them. The Champions team would be happy to do this, but we've temporarily halted our Playbook work until our outstanding PRs get some love.

There may have been some followup UX discussion on this convention I've not been a part of. I'd be happy to talk it out with the team, and happy to be convinced otherwise if that becomes the consensus.

Reply:
Makes sense